### DISCUSSION TITLE
how does  introspectBody  like ?

### DISCUSSION THREAD

**Discussion by leehomeok** [panva/openid-client#600] [Q&A]
I called the introspect interface, but only got {active: true} ，
How can I obtain other response parameters?
For example, exp ，sid

![image](https://github.com/panva/node-openid-client/assets/13464485/c84c8b94-7d6b-424f-a057-54dcc39b1f1f)

**Comment by panva** [comment]
If you only got active than that's the only thing the introspection endpoint returned.
